Sources/Recycling/ListViews/UserAds/Cell/UserAdsListEmphasizedActionCell.swift

```diff
@@ -334,16 +351,52 @@ public class UserAdsListEmphasizedActionCell: UITableViewCell {
             self?.adImageView.backgroundColor = .clear
             self?.adImageView.image = image ?? self?.defaultImage
         }
-
     }
 
+    // MARK: - Selectors
+
     @objc private func buttonTapped(_ sender: Button) {
         delegate?.userAdsListEmphasizedActionCell(self, buttonWasTapped: sender)
     }
 
     @objc private func cancelButtonTapped(_ sender: Button) {
         delegate?.userAdsListEmphasizedActionCell(self, cancelButtonWasTapped: sender)
     }
+
+    // MARK: - Public
+
+    public func showRatingView(_ show: Bool, completion: (() -> Void)? = nil) {
```

Comment:
Would it be an idea to split this into two separate functions? `showRatingView(completion:)` / `hideRatingView(completion:)` ?  I think it would indicate what will happen clearer than `showRatingView(false)`.

Reply:
Agree 👍